apps/docs/components/blocks/NavbarTop.md

```diff
@@ -34,3 +34,33 @@ hideToc: true
 <!-- end react -->
 
 </Showcase>
+
+## NavbarTop with white background and simple mobile bar
+
+The alternate NavbarTop variant designed to work seamlessly with the NavbarBottom, providing a cohesive mobile navigation experience.
+
+By combining with the NavbarBottom, you can create a unified navigation structure that accommodates both top and bottom navigation elements on mobile devices. This ensures a smooth and intuitive user experience, allowing users to access important navigation options easily. 
+
+<Showcase showcase-name="NavbarTop/NavbarTopSimpleMobile" no-paddings style="min-height: 500px;">
```

Comment:
Question: Should these examples be so tall?

Reply:
probably not :)

Reply:
@FRSgit https://github.com/vuestorefront/storefront-ui/pull/2830/commits/b7c44b6494d0e7cb0c9d056b078cc782af0a05ea